Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -58,8 +58,20 @@ internal extension Bundle {
     static var unitTests: Bundle? {
         .init(identifier: "jp.co.rakuten.inappmessaging.Tests") ?? .init(identifier: "Tests")
     }
+    
+    static var rmcResources: Bundle? {
+        .init(identifier: "RMC-RMC-resources")
+    }
 
     private static var sdk: Bundle? {
         sdkBundle(name: "RInAppMessaging")
     }
+    
+    static var rmcSdkVersion: String? {
```

Comment:
@SoumenRautray In order to inject Bundle.main instance in GetConfigRequest and call instance function `rmcSdkVersion `, please remove `static`.

Reply:
this can be even an instance function `getRMCSdkVersion() -> String?` to be called like this - `Bundle.rmcResources?.rmcSdkVersion`. (Of course this function could be called with Bundle.main but then it will just return nil)